docs/relying-parties/how-twos/google-iap.md

```diff
@@ -0,0 +1,109 @@
+
+### Getting the current subscription status for a FxA user
+
+Assuming you are already integrated with FxA generally (see [Prerequisites](#prerequisites)), are receiving subscription updates for other types of subscriptions (e.g. for PayPal or Stripe), and you have completed the [mapping of Play Store to Stripe plans](#map-google-iap-to-stripe-plans), no further work is required to continue receiving these updates for Google IAP subscribers. You can also pull this information from FxA with the existing [`/account/profile`](/ecosystem-platform/api#tag/Account/operation/getAccountProfile) endpoint without any further changes.
+
+### Registering a Google IAP subscriber
+
+SubPlat needs to be able to associate a Google IAP subscription in Google's system to a specific FxA user. This is done by registering each Google IAP subscriber with the [`/subscriptions/iap/play-token/${appName}`](/ecosystem-platform/api#tag/Subscriptions/operation/postOauthSubscriptionsIapPlaytokenAppname) endpoint.
+
+By design, we allow only one FxA user per Play Store [`token`](https://developer.android.com/google/play/billing#tokens), which is how Google uniquely identifies the user's device when the subscription was purchased.
+
+This endpoint can be used to register a new Google IAP subscriber or to migrate an existing Google IAP subscriber. For the latter, see [(Optional) Migrate existing Google IAP subscribers to SubPlat](#optional-migrating-existing-google-iap-subscribers-to-subplat).
+
+### (Optional) Migrating existing Google IAP subscribers to SubPlat
+
+This section only applies to products that currently have a direct integration with Google who need to migrate existing subscribers to SubPlat.
+
+This step can be done by a developer. Before proceeding, complete the steps under [Configuration](#configuration) and [SubPlat API Calls](#subplat-api-calls). See also the [Testing your integration](#testing-your-integration) section.
+
+To migrate existing subscribers to SubPlat, use the existing registration endpoint ([`/subscriptions/iap/play-token/${appName}`](/ecosystem-platform/api#tag/Subscriptions/operation/postOauthSubscriptionsIapPlaytokenAppname)) to register each existing user with SubPlat.
+
+:::note
+
+While SubPlat routes can handle thousands of requests per second, we recommend limiting requests to 10-20 per minute to start and potentially increase that if there are no issues. This is because, at the time of writing, the Google Play Developer API docs states a default rate limit threshold of 200,000 queries per day, and we make more than one request to Google for each SubPlat request.
+
+:::
+
+## Testing your integration
+
+:::caution
+Like many systems, Google's Sandbox environment does not necessarily exhibit behavior consistent with what is stated in their docs. When in doubt, directly test the scenario in question, potentially more than once, to understand what to expect.
```

Comment:
I'm not sure if Google has a separate Sandbox environment like Apple. I'd look at the source code to see how we configure API calls for Google -- is there a separate Sandbox versus Production URL? Does Google even use that terminology? You may need to update the terminology here and references to a Sandbox.